config/initializers/statsd.rb

```diff
@@ -63,6 +63,34 @@
     end
   end
 
+  # Sign in Service
+  SignInController::REDIRECT_URLS.each do |csp|
+    StatsD.increment(SignInController::STATSD_SIS_AUTHORIZE_ATTEMPT_SUCCESS, 0,
+                     tags: ['version:v0', "context:#{csp}"])
+    StatsD.increment(SignInController::STATSD_SIS_AUTHORIZE_ATTEMPT_FAILURE, 0,
+                     tags: ['version:v0', "context:#{csp}", 'error:'])
+    StatsD.increment(SignInController::STATSD_SIS_CALLBACK_SUCCESS, 0,
+                     tags: ['version:v0', "context:#{csp}"])
+    StatsD.increment(SignInController::STATSD_SIS_CALLBACK_FAILURE, 0,
+                     tags: ['version:v0', "context:#{csp}", 'error:'])
+    StatsD.increment(SignInController::STATSD_SIS_TOKEN_SUCCESS, 0,
+                     tags: ['version:v0'])
+    StatsD.increment(SignInController::STATSD_SIS_TOKEN_FAILURE, 0,
+                     tags: ['version:v0', 'error:'])
```

Comment:
The actual errors for these will be defined in the main PR, right? Just confirming, otherwise looks good

Reply:
Correct.